app/src/main/java/com/nearbynest/app/data/remote/ImageApiService.kt

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright COPYRIGHT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nearbynest.app.data.remote
+
+import com.nearbynest.app.data.remote.apiresult.ApiResult
+import com.nearbynest.app.data.remote.response.PropertyImageResponse
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ImageApiService {
+    @GET("/api/")
+    suspend fun getImages(
+        @Query("key") apiKey: String,
```

Comment:
Interesting, if you pass API key as query then it's clearly visible to any user. It's not safe way to use. Can't you add it as a header?

Reply:
I can not put 'Key' in header because in documentation of api, they have mentioned to use it as query.
[https://pixabay.com/api/docs/](url)